.github/workflows/main.yml

```diff
@@ -143,11 +127,15 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
+      ### END runner setup
+
       - name: Jackson Integration Tests ${{ matrix.target-library }}
         run: mvn verify -pl :nessie-client -am -Pjackson-tests -Djackson.test.version=${{ matrix.jackson-version }} -Dtest.log.level=WARN
+
   python:
     name: Python
     runs-on: ubuntu-latest
+    if: ${{ github.repository == 'projectnessie/nessie' }}
```

Comment:
im confused about this line. We have added this in a few places and removed it in others. What is the limitation>

Reply:
It's more a convenience thing for nessie-forks, so that workflow-runs on forks cannot complain about missing secrets.
In `main.yml` (as in this PR), it's not needed, because it doesn't use any secrets.

Reply:
Removed the changes to `main.yml`